lib/bolt/node/winrm.rb

```diff
@@ -92,10 +107,22 @@ def shell_init
   # streams must have .ReadToEnd() called prior to process .WaitForExit()
   # to prevent deadlocks per MSDN
   # https://msdn.microsoft.com/en-us/library/system.diagnostics.process.standarderror(v=vs.110).aspx#Anchor_2
-  $process.StandardOutput.ReadToEnd() | Out-Host
```

Comment:
In the previous iteration, performing synchronous reads on *both* streams has the potential to deadlock.